cfgov/v1/models/base.py

```diff
@@ -131,6 +131,14 @@ class CFGOVPage(Page):
         ObjectList(settings_panels, heading='Configuration'),
     ])
 
+    def get_authors(self):
```

Comment:
I realize this function isn't totally necessary but it was a design decision, in case we ever want to add sort_by parameters or another default.